PetGame.Core/Models/LeaderboardEntry.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace PetGame.Models
+{
+    /// <summary>
+    /// A type that represents one entry (row) in the leaderboard
+    /// </summary>
+    public class LeaderboardEntry
+    {
+        public int Score { get; set; }
+        public DateTime Timestamp { get; set; }
```

Comment:
See #44 for how to add `JsonConverterAttribute`, `JsonPropertyAttribute`, and `JsonObjectAttribute` attributes.